pom.xml

```diff
@@ -231,6 +232,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-source-plugin</artifactId>
+                <version>3.0.1</version>
```

Comment:
Is this change here to eliminate the warning we see on each Maven build?

Reply:
yes I just merge PR #2689 here